app/src/main/java/com/example/evently/EntrantListAdapter.java

```diff
@@ -0,0 +1,62 @@
+package com.example.evently;
+
+import java.util.ArrayList;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.example.evently.data.model.Account;
+
+/**
+ * Recycler view that displays each entrant as a row with their profile picture and name
+ */
+public class EntrantListAdapter extends RecyclerView.Adapter<EntrantListAdapter.ViewHolder> {
+    private ArrayList<Account> entrants;
+    private LayoutInflater inflater;
+
+    EntrantListAdapter(Context context, ArrayList<Account> entrants) {
+        this.entrants = entrants;
+        this.inflater = LayoutInflater.from(context);
+    }
+
+    public static class ViewHolder extends RecyclerView.ViewHolder {
+        TextView image;
+        TextView name;
+
+        public ViewHolder(View view) {
+            super(view);
+            // Can define click listeners here
+
+            this.image = view.findViewById(R.id.image_placeholder);
+            this.name = view.findViewById(R.id.entrant_name);
+        }
+    }
+
+    @NonNull @Override
+    public EntrantListAdapter.ViewHolder onCreateViewHolder(
+            @NonNull ViewGroup parent, int viewType) {
+        View view = inflater.inflate(R.layout.event_entrants_list_content, parent, false);
```

Comment:
There should be a bound class for this layout. At the moment, it's probably called `EventEntrantsListContentBinding`.

I'd highly suggest renaming the `event_entrants_list_content.xml` layout. It's quite long yet not descriptive. `fragment_event_entrant.xml`  would be better. After that change, you should be able to access its binding using `FragmentEventEntrantBinding`